workload/schemachanger: temporarily skip DROP TRIGGER in RSW

spilchen opened the pull request:
In the random schemachanger workload, you can hit an infinite recursion case when doing a cascading drop with the legacy schema changer. This root cause of this is that DROP TRIGGER can end up orphaning a backref. The fix for the root cause will be addressed in #147981. Until that is fixed, this change will skip DROP TRIGGER so we avoid the infinite recursion.

Fixes #147514

Epic: none
Release note: none

Dedej-Bergin reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/147988#-:-OSK7F768T5DkBP7al1Y:bnfp4nl)* status: <span>:shipit:</span> complete! 1 of 0 LGTMs obtained (waiting on @DarrylWong and @herkolategan)



<!-- Sent from Reviewable.io -->

spilchen committed:
workload/schemachanger: temporarily skip DROP TRIGGER in RSW

In the random schemachanger workload, you can hit an infinite recursion
case when doing a cascading drop with the legacy schema changer. This
root cause of this is that DROP TRIGGER can end up orphaning a backref.
The fix for the root cause will be addressed in #147981. Until that is
fixed, this change will skip DROP TRIGGER so we avoid the infinite
recursion.

Fixes #147514

Epic: none
Release note: none
